elasticsearch-model/gemfiles/3.0.gemfile

```diff
@@ -9,4 +9,4 @@ gemspec path: '../'
 
 gem 'activemodel',  '>= 3.0'
 gem 'activerecord', '~> 3.2'
-gem 'mongoid',      '>= 3.0'
+gem 'mongoid',      '~> 3.0'
```

Comment:
Hmm, an you please provide reasoning for the change, in commit message or via comment? 

I'd like to keep the gemfiles intact, since it frequently totally messes up everything and I have to spend hours debugging it.